tests/queries/0_stateless/02435_rollback_cancelled_queries.sh

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env bash
-# Tags: no-random-settings, no-ordinary-database, no-fasttest
+# Tags: no-random-settings, no-ordinary-database, no-fasttest, no-azure-blob-storage
```

Comment:
Maybe `no-parallel`?

Reply:
I'd rather just exclude azure tbh. The 30 seconds timeout might be too tight even when running non-parallel sometimes, and I don't want this test to become flaky again because of the same reason. And at the same time I don't want to increase the timeout just because of Azure, and there's no perfect timeout anyway (90 seconds sounds safe for now, but maybe still not enough). And also this test doesn't seem to interfere with others in any other aspect, so not enough reasoning for making it non-parallel imho